packages/ssi-types/src/mapper/credential-mapper.ts

```diff
@@ -293,6 +293,82 @@ export class CredentialMapper {
     return '@context' in credential && ((credential as ICredential).type?.includes('VerifiableCredential') || false)
   }
 
+  public static isCredential(original: OriginalVerifiableCredential | OriginalVerifiablePresentation): boolean {
+    let document = original
+    try {
+      if (typeof original === 'string') {
+        if (this.isJwtEncoded(original)) {
+          const vc: IVerifiableCredential = this.toUniformCredential(original)
+          return this.isW3cCredential(vc)
+        } else if (this.isSdJwtEncoded(original)) {
+          // find out how we can differentiate between vc and vp for sd-jwts
```

Comment:
Remove this comment, because they are always a credential

Reply:
removed